includes/event/event-cpt.php

```diff
@@ -46,13 +46,14 @@ function create_event_cpt()
         'menu_position' => 5,
         'show_in_admin_bar' => true,
         'show_in_nav_menus' => true,
+        'show_in_rest' => true,
         'can_export' => true,
         'has_archive' => true,
         'exclude_from_search' => false,
         'publicly_queryable' => true,
         'capability_type' => 'post',
         'menu_icon' => 'dashicons-calendar-alt',
     );
-    register_post_type('event', $args);
+    register_post_type('events', $args);
```

Comment:
Post types are better named in singular form, yet we can rewrite their slug in plural
If interested how, read this [short article](https://www.tannerrecord.com/custom-post-type-naming/)

Reply:
```suggestion
    register_post_type('event', $args);
```